rrmngmnt/filesystem.py

```diff
@@ -30,3 +30,7 @@ def rmdir(self, path):
         return self.host.executor().run_cmd(
             ['rm', '-rf', path]
         )[0] == 0
+
+    def is_dir_empty(self, path):
```

Comment:
I would rather call is listdir() it covers more cases of usage.
by 'ls' you bring all data back so why not to process them?

then you can use it in predicate like:

if not host.fs.listdir('/path/to/dir'):
    pass